src/scss/override/_forms.scss

```diff
@@ -173,96 +173,139 @@ select[multiple].form-control {
   }
-}
 
-.checkbox input[type=checkbox] {
-  &::-ms-check {
-    color: $win-color-type-primary;
-    border-color: $win-color-fl-60;
-    background-color: transparent;
+  .form-group-label {
+    font-size: 15px;
+    font-weight: 400;
+    margin-bottom: 0;
+    line-height: 1em;
   }
 
-  &:indeterminate::-ms-check {
-    color: $win-color-type-primary;
-    border-color: $win-color-fl-accent;
-  }
+  margin-bottom: 36px;
 
-  &:checked::-ms-check {
-    color: $win-color-light;
-    background-color: $win-color-fl-accent;
-    border-color: transparent;
-  }
+}
 
-  &:hover::-ms-check {
-    border-color: $win-color-type-primary;
+// Common checkbox and radio styles
+.checkbox,
+.radio {
+  margin: 24px 0;
+
+  & input[type=checkbox],
+  & input[type=radio] {
+    width: 20px;
+    height: 20px;
+    margin-left: -28px;
+    margin-top: 0;
   }
 
-  &:hover:indeterminate::-ms-check {
-    color: $win-color-type-primary;
-    border-color: $win-color-fl-accent;
-  }
+  & label {
+    padding-left: 0px;
+    font-weight: 400;
+    margin-left: 28px;
+    min-height: initial;
+    line-height: 20px;
+    input[type=checkbox],
+    input[type=radio] {
+        opacity: 0;
+    }
 
-  &:active::-ms-check {
-    border-color: transparent;
-    background-color: $win-color-fl-50;
-  }
+    span::before {
+        border-width: 1px;
+        border-style: solid;
+        border-color: $win-color-fl-60;
+        content: "";
+        height: 20px;
+        width: 20px;
+        position: absolute;
+        left: 0;
+        cursor: pointer;
+    }
 
-  &:indeterminate:active::-ms-check {
-    color: $win-color-type-secondary;
-    border-color: $win-color-fl-60;
-    background-color: transparent;
-  }
+    input:disabled {
+      ~ span {
+        color: $win-color-type-disabled;
+      }
 
-  &:disabled::-ms-check, &:indeterminate:disabled::-ms-check {
-    color: $win-color-type-disabled-alt;
-    border-color: $win-color-fl-30;
-    background-color: transparent;
+      ~ span::before {
+        border-color: $win-color-fl-25;
+        color: $win-color-type-disabled;
+      }
+    }
   }
 }
 
-.radio input[type=radio] {
-  &::-ms-check {
-    color: $win-color-type-primary;
-    border-color: $win-color-fl-60;
-    background-color: transparent;
-  }
+// Checkbox only
+.checkbox {
+
+  & label {
+    input:checked ~ span::before {
+        background-color: $win-color-fl-accent;
+        border-color: $win-color-fl-accent;
+        color: $win-color-fl-70-alt;
+        font-family: $win-font-symbols;
+        content: "\E081";
+        text-indent: 2px;
+        line-height: 1em;
+        padding-top: 2px;
+    }
 
-  &:hover::-ms-check {
-    border-color: $win-color-type-primary;
+    input:indeterminate ~ span::before {
+      border-color: $win-color-fl-accent;
+      color: $win-color-fl-60;
+      content: "■"; // Use "\E73C" when we get Basel Symbols font
+      text-indent: 2px;
+      line-height: 12px;
+      font-size: 17px;
```

Comment:
The font-size here is sizing the indeterminate square inside the the checkbox. It stays fixed across viewports.